lib/charms/opensearch/v0/opensearch_relation_peer_cluster.py

```diff
@@ -172,6 +174,7 @@ def _on_peer_cluster_relation_changed(self, event: RelationChangedEvent):
         if orchestrators.failover_app and orchestrators.failover_rel_id in target_relation_ids:
             logger.info("A failover cluster orchestrator is already registered.")
             self.refresh_relation_data(event)
+            event.defer()
```

Comment:
why is this needed? this condition simply means that the failover candidacy got rejected from a candidate, because one was already registered